src/components/Layout/Colors.module.scss

```diff
@@ -0,0 +1,5 @@
+$base-color: #FFCD1B;
```

Comment:
Configuration SCSS files (like this one) could just go to a separate scss file that won't be compiled by Sass (by adding `_` in front of the filename).

I keep those in my `src/styles` directory and import them in other Scss modules as such:

```scss
// example: src/styles/_colors.scss

$button-color: #9900aa;
```

```scss
// example: src/components/Button/Button.module.scss

@import 'src/styles/colors'; // note you don't have to include the _ symbol in the import here

.button {
    color: $button-color;
}
```